Cargo.toml

```diff
@@ -16,12 +16,17 @@ keywords = [ "crypto", "cryptography", "allocator" ]
 
 [dependencies]
 libc = '0'
+libsodium-sys = { version = '0.2', optional = true }
 
 [build-dependencies]
 pkg-config = '0.3'
 
 [dev-dependencies]
-libsodium-sys = { version = '0.2', features = ['use-pkg-config'] }
+libsodium-sys = { version = '0.2' }
 
 [features]
-allow-coredumps = []
+default         = ["use-pkg-config"]
```

Comment:
The problem of having `use-pkg-config` as a feature is that if a 3rd party dependency has just `secrets = "1"` the the final executable with also get `use-pkg-config`, even if `default-features = false` is set.

Cargo is creating a set of the feature gates from all dependencies. I believe the correct way is having only `use-libsodium-sys` and disabled.

Reply:
Ah, good to know. I'll make that change and fix some of the resulting warnings and get this merged today.